### PR TITLE
feat: news in app [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by brainbicycle**
This PR resolves [] 

### Description

Adds an news component to the home feed below the fold above editorial and a news screen that shows all news articles.

https://github.com/artsy/eigen/assets/49686530/9d6a704f-45d2-4ad6-9e70-8935e203f06e

Follow-ups (if we want to ship):
- Fix tracking copied over from editorial
- Tests
- Feature flag
- Review with design and product if this is something we want and what we want changed 

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Home feed news module and news screen - Brian 

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c5540b451c191a42ea4d704d7a728037f0114148
</p>

File: src/app/Scenes/Article/ArticleScreen.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const Article: React.FC<ArticleScreenProps> = (props) => {
     return null
   }
 
-  const NATIVE_LAYOUTS = ["STANDARD", "FEATURE"]
```
Comment (brainbicycle) [inline src/app/Scenes/Article/ArticleScreen.tsx:42]: I just added this as a supported layout and it seems to mostly just work but probably warrants some more testing

File: src/app/Scenes/Home/useHomeModules.ts [hunk 2]
```diff
@@ -102,6 +103,17 @@ export const useHomeModules = (props: HomeProps) => {
         type: "galleriesForYouBanner",
         hidden: !enableGalleriesForYou,
       },
+      {
+        contextModule: ContextModule.articleRail,
```
Comment (brainbicycle) [inline src/app/Scenes/Home/useHomeModules.ts:107]: todo: fix tracking, just copied from editorial

File: src/app/Scenes/Articles/News/ArticlesNews.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  }
+}
+
+export interface ArticleNewsProps {
+  viewer: {
+    articles: NewsArticle[]
+  }
+}
+
+// TODO: Rename this component to something like NewsCard, relay blew up earlier when I tried
```
Comment (brainbicycle) [inline src/app/Scenes/Articles/News/ArticlesNews.tsx:21]: todo rename it

**Comment by araujobarret** [comment]
@brainbicycle I think we can close this one, thanks for the work here, as mentioned in the linked PR it was extracted and adapted 👍

**Pull request closed without merging**